src/bot.js

```diff
@@ -0,0 +1,99 @@
+import { PIXELS_PER_TILE } from './constants';
+import { getTraversableCells } from './helpers/map';
+import {
+  findMyEndzone,
+  findEnemyEndzone,
+  findFlagStation,
+  findEnemy,
+  findEnemyFC,
+} from './helpers/finders';
+import { myTeamHasFlag, enemyTeamHasFlag } from './helpers/gameState';
+import { getMe } from './helpers/player';
+import getTarget from './helpers/path';
+import { move } from './utils';
+
+/*
+ * The logic/flowchart to get where our goal is.
+ *   If I have the flag, go to my endzone.
+ *   If an enemy in view has the flag, chase him.
+ *   If the enemy team has the flag but I can't see them, go to their endzone.
+ *   If we have the flag, go to our endzone.
+ *   Else, go to the flag station.
+ */
+function getGoal() {
+  let goal;
+  const me = getMe();
+  // If the bot has the flag, go to the endzone
+  if (me.flag) {
+    const chaser = findEnemy();
+    if (false && chaser) { // eslint-disable-line no-constant-condition
+      // Really bad jukes !!!!! DISABLED FOR NOW
```

Comment:
it's disabled because it's inside of an `if (false)` statement. Yeah I assume someone tried to do jukes and it was bad but wanted to keep it in case we want to update it later.  I could comment it out?